File: golang-backend/internal/model/request_log.go
package model

type RequestLog struct {
	ID           int64   `gorm:"primaryKey;autoIncrement" json:"id"`
	Method       string  `gorm:"not null;index:idx_request_log_method_time" json:"method"`
	URL          string  `gorm:"not null" json:"url"`
	RouteName    *string `gorm:"column:route_name" json:"routeName"`
	Module       *string `json:"module"`
	StatusCode   int     `gorm:"column:status_code;index:idx_request_log_status_time" json:"statusCode"`
	RequestBody  *string `gorm:"column:request_body" json:"requestBody"`
	ResponseBody *string `gorm:"column:response_body" json:"responseBody"`
	Duration     int     `json:"duration"`
	Timestamp    int64   `gorm:"not null;index:idx_request_log_method_time;index:idx_request_log_status_time;index:idx_request_log_time" json:"timestamp"`
	CreatedAt    int64   `gorm:"column:created_at" json:"createdAt"`
}

func (RequestLog) TableName() string { return "request_logs" }

File: python-backend/app/models/request_log.py
"""RequestLog ORM model."""

from sqlalchemy import BigInteger, Index, Integer, String
from sqlalchemy.orm import Mapped, mapped_column

from app.core.database import Base


class RequestLog(Base):
    __tablename__ = "request_logs"

    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
    method: Mapped[str] = mapped_column(String, nullable=False)
    url: Mapped[str] = mapped_column(String, nullable=False)
    route_name: Mapped[str | None] = mapped_column(String, nullable=True)
    module: Mapped[str | None] = mapped_column(String, nullable=True)
    status_code: Mapped[int] = mapped_column(Integer, nullable=False)
    request_body: Mapped[str | None] = mapped_column(String, nullable=True)
    response_body: Mapped[str | None] = mapped_column(String, nullable=True)
    duration: Mapped[int] = mapped_column(Integer, nullable=False)
    timestamp: Mapped[int] = mapped_column(BigInteger, nullable=False)
    created_at: Mapped[int] = mapped_column(BigInteger, nullable=False)

    __table_args__ = (
        Index("idx_request_log_method_time", "method", "timestamp"),
        Index("idx_request_log_status_time", "status_code", "timestamp"),
        Index("idx_request_log_module_time", "module", "timestamp"),
        Index("idx_request_log_time", "timestamp"),
    )
